<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>530</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="mainwindow.qrc">
    <normaloff>:/res/appicon.png</normaloff>:/res/appicon.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>128</width>
    <height>128</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>0</y>
      <width>718</width>
      <height>444</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="dictTab">
     <attribute name="title">
      <string>dictionary</string>
     </attribute>
     <widget class="QListView" name="indexListView">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>160</width>
        <height>340</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputLineEdit">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>14</y>
        <width>190</width>
        <height>32</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="srcLanComboBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>14</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="detLanComboBox">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>14</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>289</x>
        <y>21</y>
        <width>32</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="mainwindow.qrc">:/res/to_right_arrow.png</pixmap>
      </property>
     </widget>
     <widget class="QTextEdit" name="dictTextEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>69</y>
        <width>540</width>
        <height>271</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="pgupToolButton2">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>378</y>
        <width>24</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="mainwindow.qrc">
        <normaloff>:/res/left_arrow2.png</normaloff>:/res/left_arrow2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>18</width>
        <height>32</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
      <property name="arrowType">
       <enum>Qt::NoArrow</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="pgupToolButton1">
      <property name="geometry">
       <rect>
        <x>47</x>
        <y>378</y>
        <width>24</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="mainwindow.qrc">
        <normaloff>:/res/left_arrow.png</normaloff>:/res/left_arrow.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>18</width>
        <height>32</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="pgdownToolButton1">
      <property name="geometry">
       <rect>
        <x>87</x>
        <y>378</y>
        <width>24</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="mainwindow.qrc">
        <normaloff>:/res/right_arrow.png</normaloff>:/res/right_arrow.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>18</width>
        <height>32</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="pgdownToolButton2">
      <property name="geometry">
       <rect>
        <x>126</x>
        <y>378</y>
        <width>24</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="mainwindow.qrc">
        <normaloff>:/res/right_arrow2.png</normaloff>:/res/right_arrow2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>18</width>
        <height>32</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="indexLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>349</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>filter by a prefix</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="saveButton">
      <property name="geometry">
       <rect>
        <x>572</x>
        <y>356</y>
        <width>48</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="mainwindow.qrc">
        <normaloff>:/res/favorite.png</normaloff>:/res/favorite.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="netdictButton">
      <property name="geometry">
       <rect>
        <x>641</x>
        <y>356</y>
        <width>48</width>
        <height>48</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>No action</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="mainwindow.qrc">
        <normaloff>:/res/web.png</normaloff>:/res/web.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="queryButton">
      <property name="geometry">
       <rect>
        <x>654</x>
        <y>6</y>
        <width>48</width>
        <height>48</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="mainwindow.qrc">
        <normaloff>:/res/search.png</normaloff>:/res/search.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settingTab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>setting</string>
     </attribute>
     <widget class="QTabWidget" name="settingTabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-3</y>
        <width>717</width>
        <height>414</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>general</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>8</y>
          <width>244</width>
          <height>100</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#groupBox_3 { 
     border: 1px solid gray; 
     border-radius: 1px; 
 } </string>
        </property>
        <property name="title">
         <string>option</string>
        </property>
        <widget class="QLabel" name="uilanLabel">
         <property name="geometry">
          <rect>
           <x>17</x>
           <y>37</y>
           <width>90</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>UI Language</string>
         </property>
        </widget>
        <widget class="QComboBox" name="uilanComboBox">
         <property name="geometry">
          <rect>
           <x>112</x>
           <y>32</y>
           <width>90</width>
           <height>27</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>English</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>汉语</string>
          </property>
         </item>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>dict</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>10</y>
          <width>390</width>
          <height>101</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#groupBox { 
     border: 1px solid gray; 
     border-radius: 1px; 
 } </string>
        </property>
        <property name="title">
         <string>capture word setting</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <widget class="QCheckBox" name="cwsSelectionCheckBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>161</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>mouse selection</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="cwsClipboardCheckBox">
         <property name="geometry">
          <rect>
           <x>219</x>
           <y>30</y>
           <width>161</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>clipboard</string>
         </property>
        </widget>
       </widget>
       <widget class="QListWidget" name="dictListWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>261</width>
          <height>300</height>
         </rect>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::NoDragDrop</enum>
        </property>
       </widget>
       <widget class="QToolButton" name="dictDownToolButton">
        <property name="geometry">
         <rect>
          <x>185</x>
          <y>329</y>
          <width>59</width>
          <height>30</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="mainwindow.qrc">
          <normaloff>:/res/down.png</normaloff>:/res/down.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QToolButton" name="dictUpToolButton">
        <property name="geometry">
         <rect>
          <x>49</x>
          <y>329</y>
          <width>59</width>
          <height>30</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="mainwindow.qrc">
          <normaloff>:/res/up.png</normaloff>:/res/up.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="vocabularyTab">
     <attribute name="title">
      <string>vbook</string>
     </attribute>
     <widget class="QTabWidget" name="vocabularyTabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-3</y>
        <width>717</width>
        <height>411</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="listModeTable">
       <attribute name="title">
        <string>list</string>
       </attribute>
       <widget class="QToolButton" name="vbdelToolButton">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>321</y>
          <width>32</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="mainwindow.qrc">
          <normaloff>:/res/remove.png</normaloff>:/res/remove.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QToolButton" name="vbclearToolButton">
        <property name="geometry">
         <rect>
          <x>175</x>
          <y>322</y>
          <width>32</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="mainwindow.qrc">
          <normaloff>:/res/clear_all.png</normaloff>:/res/clear_all.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="vbexplTextEdit">
        <property name="geometry">
         <rect>
          <x>306</x>
          <y>36</y>
          <width>368</width>
          <height>240</height>
         </rect>
        </property>
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QListView" name="vbookListView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>281</width>
          <height>290</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="vbModeTab">
       <attribute name="title">
        <string>exam</string>
       </attribute>
       <widget class="QToolButton" name="vbpreItemTlBtn">
        <property name="geometry">
         <rect>
          <x>109</x>
          <y>187</y>
          <width>24</width>
          <height>52</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="mainwindow.qrc">
          <normaloff>:/res/left_arrow.png</normaloff>:/res/left_arrow.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QToolButton" name="vbnextItemTlBtn">
        <property name="geometry">
         <rect>
          <x>541</x>
          <y>187</y>
          <width>24</width>
          <height>52</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="mainwindow.qrc">
          <normaloff>:/res/right_arrow.png</normaloff>:/res/right_arrow.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="vbInput">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>39</y>
          <width>270</width>
          <height>27</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
       </widget>
       <widget class="QLabel" name="vbScoreLabel">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>50</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>90</y>
          <width>350</width>
          <height>250</height>
         </rect>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>348</width>
           <height>248</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="vbExplLabel">
            <property name="text">
             <string/>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSetting"/>
   <addaction name="actionVocabulary"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionVocabulary">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/vocabulary.png</normaloff>:/res/vocabulary.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/res/setting.png</normaloff>:/res/setting.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="toolTip">
    <string>setting</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
